crates/hive-utils/src/madara/utils.rs

```diff
@@ -172,9 +173,17 @@ mod tests {
     use starknet_api::core::{ClassHash, ContractAddress as StarknetContractAddress, Nonce};
     use starknet_api::hash::StarkFelt;
     use starknet_api::state::StorageKey as StarknetStorageKey;
+    use tracing_subscriber::{filter, FmtSubscriber};
 
     use super::*;
 
+    #[ctor]
```

Comment:
adding ctor to help in debug

Reply:
according to the doc, one should not add subscribers in a lib, see https://docs.rs/tracing/0.1.37/tracing/#in-libraries

do you want to add this for the tests? or for the `dump-katana` bin? you should place this subscriber in the target location then

Reply:
https://github.com/kkrt-labs/kakarot-rpc/blob/bdd0c79aa6850d140d8ff4285b3f1b3885a6db6d/crates/core/tests/client.rs#L26-L31

Yeah similar to the tests in the client, I added it to here to make debugging the tests easier. It's how I was able to find how we're managing the lock. From my understanding, adding it inside the test module is acceptable since it will only affect the test execution environment and won't impact the rpc application

Reply:
> according to the doc, one should not add subscribers in a lib, see https://docs.rs/tracing/0.1.37/tracing/#in-libraries
> 
> do you want to add this for the tests? or for the `dump-katana` bin? you should place this subscriber in the target location then

Are you saying it's better to use ctor in the test module where we import those utils?